aspnetcore/razor-components/index.md

```diff
@@ -1,65 +1,34 @@
-* **Wide browser support**: Razor Components render the UI as ordinary markup and JavaScript. Blazor runs on .NET in the browser using open web standards with no plugins and no code transpilation. Blazor works in all modern web browsers, including mobile browsers.
-
-## Hosting models
-
-### Server-side hosting model
-
-Because Razor Components decouple a component's rendering logic from how the UI updates are applied, there is flexibility in how Razor Components can be hosted. ASP.NET Core Razor Components in .NET Core 3.0 adds support for hosting Razor Components on the server in an ASP.NET Core app where all UI updates are handled over a SignalR connection. The runtime handles sending UI events from the browser to the server and then applies UI updates sent by the server back to the browser after running the components. The same connection is also used to handle JavaScript interop calls.
-
-![Razor Components runs .NET code on the server and interacts with the Document Object Model on the client over a SignalR connection](index/_static/aspnet-core-razor-components.png)
-
-For more information, see <xref:razor-components/hosting-models#server-side-hosting-model>.
-
-### Client-side hosting model
-
-Running .NET code inside web browsers is made possible by a relatively new technology, [WebAssembly](http://webassembly.org) (abbreviated *wasm*). WebAssembly is an open web standard and is supported in web browsers without plugins. WebAssembly is a compact bytecode format optimized for fast download and maximum execution speed.
-
-WebAssembly code can access the full functionality of the browser via JavaScript interop. At the same time, WebAssembly code runs in the same trusted sandbox as JavaScript to prevent malicious actions on the client machine.
-
-![Blazor runs .NET code in the browser with WebAssembly.](index/_static/blazor.png)
-
-When a Blazor app is built and run in a browser:
-
-1. C# code files and Razor files are compiled into .NET assemblies.
-1. The assemblies and the .NET runtime are downloaded to the browser.
-1. Blazor uses JavaScript to bootstrap the .NET runtime and configures the runtime to load required assembly references. Document object model (DOM) manipulation and browser API calls are handled by the Blazor runtime via JavaScript interoperability.
-
-To support older browsers that don't support WebAssembly, you can use the ASP.NET Core Razor Components [server-side hosting model](#server-side-hosting-model).
-
-For more information, see <xref:razor-components/hosting-models#client-side-hosting-model>.
+* Build rich interactive UIs using .NET instead of JavaScript.
+* Write client-side code and share server and client-side logic among apps.
+* Razor Components render the UI as HTML markup and JavaScript for wide browser support, including mobile browsers.
```

Comment:
Render the UI as HTML and CSS for wide browser support, including mobile browsers.